luigi_pipeline/sex_related_checks.py

```diff
@@ -0,0 +1,568 @@
+    )
+    output_dir = luigi.Parameter(
+        description="Path to write the sex ploidy output table.",
+        default="gs://seqr-loading-temp/luigi-sex-check/",
+    )
+    temp_dir = luigi.Parameter(
+        description="Path to write the temporary output. End with '/'",
+        default="gs://seqr-scratch-temp/luigi-sex-check/",
+    )
+    genome_version = luigi.ChoiceParameter(
+        description="Reference Genome Version (37 or 38)",
+        choices=["GRCh37", "GRCh38"],
+        default="GRCh38",
+    )
+    sample_type = luigi.ChoiceParameter(
+        choices=["WGS", "WES"], description="Sample type, WGS or WES", var_type=str
+    )
+    dont_validate = luigi.BoolParameter(
+        description="Disable checking whether the dataset matches the specified "
+        "genome version and WGS vs. WES sample type."
+    )
+    remap_path = luigi.OptionalParameter(
+        default=None, description="Path to a tsv file with two columns: s and seqr_id."
+    )
+    subset_path = luigi.OptionalParameter(
+        default=None, description="Path to a tsv file with one column of sample IDs: s."
+    )
+    use_y_cov = luigi.BoolParameter(
+        description="Whether to use chromosome Y coverage when inferring sex. Note that Y coverage is required to infer sex aneuploidies."
+    )
+    add_x_cov = luigi.BoolParameter(
+        description="Whether to also calculate chromosome X mean coverage. Must be specified with use-y-cov."
+    )
+    y_cov_threshold = luigi.FloatParameter(
+        default=0.1,
+        description="Y coverage threshold used to infer sex aneuploidies (XY samples below and XX samples above this threshold will be inferred as having aneuploidies).",
+    )
+    xy_fstat_threshold = luigi.FloatParameter(
+        default=0.75,
+        description="F-stat threshold above which a sample will be called XY. Default is 0.75.",
+    )
+    xx_fstat_threshold = luigi.FloatParameter(
+        default=0.50,
+        description="F-stat threshold below which a sample will be called XX. Default is 0.50.",
+    )
+    aaf_threshold = luigi.FloatParameter(
+        default=0.05,
+        description="Alternate allele frequency threshold for `hl.impute_sex`. Default is 0.05.",
+    )
+    callrate_threshold = luigi.FloatParameter(
+        default=0.25,
+        description="Minimum variant call rate threshold. Default is 0.25.",
+    )
+    normalization_contig = luigi.Parameter(
+        default="20",
+        description="Autosome to use to normalize sex chromosome coverage. Default is chromosome 20.",
+    )
+    hail_temp_dir = luigi.OptionalParameter(
+        default="gs://seqr-scratch-temp/",
+        description="Networked temporary directory used by hail for temporary file storage. Must be a network-visible file path.",
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def output_directory(self):
```

Comment:
I think both `output_directory` and `temp_directory` don't need to be `Targets` here... which should make the calling code a little more readable.  

I think the `Target` class is generally used for the output of the task itself, so luigi will know that the task has succeeded.